### ISSUE TITLE
[FEATURE] - Bookmark for file or folder

### ISSUE THREAD

**Issue by sabahtalateh** [alefragnani/vscode-bookmarks#673]
Please add ability to add bookmark folder like it is done in this extension for example. 
https://marketplace.visualstudio.com/items?itemName=UrosVujosevic.explorer-manager

Thanks

**Comment by man-chi** [comment]
yes, I am second to this. I also wish to have a feature to support bookmark for file or folder level

**Comment by lucascosti** [comment]
Big +1 from me. I work in a lot of large repos, and repeatedly having to find/scroll to the same files or folders is a pain. It would be awesome to be able to bookmark a file or folder in a similar manner with this extension that we bookmark lines.

**Comment by alefragnani** [comment]
Hi,

With a new year starting, it is the moment to review the extension’s backlog in order to keep it healthy and focused.

I'm closing this feature request as a duplicate of #658, because if a file-level bookmark is added, it would require effectively toggling the bookmark (either in some file location or at file level), Bookmarks for folder, on the other hand, is out of scope.

Still a big Thank You to you for taking the time to create this issue!

Thanks for your understanding
